smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/CommandGenerator.java

```diff
@@ -185,8 +185,12 @@ private String getCommandExample(String serviceName, String configName, String c
                                 model.getShape(operation.getOutputShape()).get(), model, true))
                 + "\n```\n"
                 + "\n"
-                + String.format("@param %s - {@link %s}%n", commandInput, commandInput)
-                + String.format("@returns {@link %s}%n", commandOutput)
+                + String.format("@param %s - {@link %s}%n",
+                    model.getShape(operation.getInputShape()).get().getId().getName(),
```

Comment:
The name of the input is `input` in the constructor and `args` in the aggregated client methods. However, no need to change this since in both cases there is only 1 positional param.